protos/farm_ng/core/event_service.proto

```diff
@@ -0,0 +1,85 @@
+//    Copyright 2022, farm-ng inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+
+
+syntax = "proto3";
+
+import "farm_ng/core/uri.proto";
+import "farm_ng/core/timestamp.proto";
+import "farm_ng/core/event.proto";
+
+package farm_ng.core.proto;
+
+service EventService {
+    rpc getServiceState(GetServiceStateRequest) returns (GetServiceStateReply) {}
```

Comment:
I think this api could be removed, im not sure why its important to track the state of the service at this level.  I think grpc connection state can give us everything we need. @edgarriba @lullabee @Hackerman342

Reply:
it was originally added to keep tracked of the status of the services in the launcher